### DISCUSSION TITLE
How to Install a app very clean 110% wasm in a cheap host Linux

### DISCUSSION THREAD

**Discussion by RIldefonso** [unoplatform/uno#10258] [Q&A]
Do you know that type of host that you can install WordPress and PHP apps very clean?
Is it possible to install there a UNO app that use only calcs and screen events?

Thanks for help.

**Comment by jeromelaban** [comment]
You can deploy a WebAssembly Uno Platform app on any static file server. 

We generally use [Azure Static Apps](https://platform.uno/docs/articles/guides/azure-static-webapps.html), but any other can do, given that [proper mime types](https://platform.uno/docs/articles/how-to-host-a-webassembly-app.html) are served.

**Reply by RIldefonso** [reply]
I asked to my provider register the MIME types, and he did it. Then I created a Uno project and compiled it. In my PC it is runing.
I command to Vs2022 17.4 make the publish to a folder, then I copied this contend to a sub-folder at my site: **it didn't working!!!**

![image](https://user-images.githubusercontent.com/6700588/201287906-375b0fe6-b34e-49d2-ad48-915f30cf5cb2.png)

I am still waiting for the "Hello World".

**Reply by jeromelaban** [reply]
There is no need to create multiple conversation on the same topic.

By looking at your browser network tab, you'll see the following:

![image](https://user-images.githubusercontent.com/5839577/201749812-7bddb512-f4de-4451-8ec8-264c854e9ecc.png)

Make sure that all the folder files can be served.

**Reply by RIldefonso** [reply]
I have no idea about what you said.
"Browser Network Tab"???  ... What is it? Where can I find it?

**Reply by jeromelaban** [reply]
See [this documentation](https://developer.chrome.com/docs/devtools/network/) on how to use it.
